### PR TITLE
Drop interplay

### PR DESCRIPTION
**Pull request by ihostage**
As part of https://github.com/playframework/interplay/issues/150

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
-addSbtPlugin("com.typesafe.play" % "interplay"            % sys.props.getOrElse("interplay.version", "3.1.6"))
 addSbtPlugin("com.typesafe.play" % "sbt-plugin"           % sys.props.getOrElse("play.version", "2.9.0-M7"))
 addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.9.0-M7"))
 
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"    % "2.5.2")
 addSbtPlugin("com.typesafe"      % "sbt-mima-plugin" % "1.1.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"      % "5.10.0")
+
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.12")
```
Comment (ihostage) [inline project/plugins.sbt:12]: @mkurz How did we release this library without `sbt-ci-release` plugin? 🤔

**Comment by ihostage** [comment]
@mkurz I updated this PR by your comments from https://github.com/playframework/play-doc/pull/206

File: project/plugins.sbt (hunk 1 above)
Comment (mkurz) [reply project/plugins.sbt:11]: interplay provided that: https://github.com/playframework/interplay/blob/ad1bddb900e2883f7ad78e59c70fc86603903a52/build.sbt#L19

**Review by mkurz — APPROVED** [review]
LGTM
The `docs` project does get the flags applied, etc. however that should not matter anyway at this point. If we need to set some (like `-Xmigration` and similar) we can do that later if needed.

**Pull request merged**